src/policy/policy.h

```diff
@@ -102,7 +102,9 @@ bool AreInputsStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
     /**
      * Check if the transaction is over standard P2WSH resources limit:
      * 3600bytes witnessScript size, 80bytes per witness stack element, 100 witness stack elements
-     * These limits are adequate for multi-signature up to n-of-100 using OP_CHECKSIG, OP_ADD, and OP_EQUAL,
+     * These limits are adequate for multi-signature up to n-of-100 using OP_CHECKSIG, OP_ADD, and OP_EQUAL.
```

Comment:
3d0948d per codebase usage and optech style guide
```suggestion
     * These limits are adequate for multisignature up to k-of-100 using OP_CHECKSIG, OP_ADD, and OP_EQUAL.
```

Reply:
Fixed.